File: src/main/java/uk/asheiou/ashutils/command/NotEnabledCommandExecutor.java
package uk.asheiou.ashutils.command;

import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;

import uk.asheiou.ashutils.MessageSender;

public class NotEnabledCommandExecutor implements CommandExecutor {

  @Override
  public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
    MessageSender.sendMessage(sender, "This command is not enabled on this server due to a missing softdepend.");
    return true;
  }
}

File: src/main/java/xyz/aeolia/ashutils/command/NotEnabledCommandExecutor.kt
package xyz.aeolia.ashutils.command

import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import xyz.aeolia.ashutils.sender.MessageSender

class NotEnabledCommandExecutor : CommandExecutor {
  override fun onCommand(
    sender: CommandSender,
    command: Command,
    label: String,
    args: Array<out String>
  ): Boolean {
    MessageSender.sendMessage(sender, "This command is not enabled on this server due to a missing" +
            " softdepend.")
    return true
  }
}